### ISSUE TITLE
CCE node doesn't supports EulerOS 2.9 version

### ISSUE THREAD

**Issue by vineet-pruthi** [opentelekomcloud/terraform-provider-opentelekomcloud#2123]
Hi, Please check the versions supported for CCE nodes:

https://docs.otc.t-systems.com/cloud-container-engine/api-ref/apis/cluster_management/creating_a_node.html#cce-02-0242
╷
│ Error: expected os to be one of [EulerOS 2.5 CentOS 7.7], got EulerOS 2.9
│
│   with opentelekomcloud_cce_node_v3.node_1,
│   on cce.tf line 24, in resource "opentelekomcloud_cce_node_v3" "node_1":
│   24:   os        = "EulerOS 2.9"
│
╵

```
resource "opentelekomcloud_cce_cluster_v3" "cluster_1" {
  name        = var.cce_cluster_name
  description = "Create cluster"

  cluster_type           = "VirtualMachine"
  cluster_version        = "v1.23"
  flavor_id              = var.cce_flavor
  vpc_id                 = opentelekomcloud_vpc_v1.vpc_1.id
  subnet_id              = opentelekomcloud_vpc_subnet_v1.subnet_1.id
  container_network_type = "overlay_l2"
  authentication_mode    = "rbac"
}

resource "opentelekomcloud_cce_node_v3" "node_1" {
  name              = "node1"
  cluster_id        = opentelekomcloud_cce_cluster_v3.cluster_1.id
  availability_zone = "eu-de-01"

  os        = "EulerOS 2.9"
  flavor_id = "s3.xlarge.2"
  key_pair  = opentelekomcloud_compute_keypair_v2.ansible_sshkey.name

  root_volume {
    size       = 40
    volumetype = "SATA"
  }

  data_volumes {
    size       = 100
    volumetype = "SSD"
  }
}

```

**Comment by artem-lifshits** [comment]
Hello @vineet-pruthi which provider version are you using? EulerOS 2.9 should be available since `v1.31.5`.

**Comment by vineet-pruthi** [comment]
Hi @artem-lifshits,  I apologize for the delay in my response. I managed to create a CCE node with EulerOS 2.9 after upgrading to opentelekomcloud provider v1.31.5 from v1.29.x.
